tasks/sast-java-sec-check.yaml

```diff
@@ -62,7 +62,7 @@ spec:
 
         ERR_MSG="$(cat stderr.txt)"
         ERR_MSG="${ERR_MSG:-unknown}"
-        HACBS_ERROR_OUTPUT=$(jq -rc --arg date $(date +%s) --arg MSG "${ERR_MSG: 0: 3000}"
+        HACBS_ERROR_OUTPUT=$(jq -rc --arg date $(date +%s) --arg MSG "${ERR_MSG//[$'\t\r\n']: 0: 3000}"
```

Comment:
is it working?
```
$ ERR_MSG='my
> cool
> error
> '
$ echo "$ERR_MSG"
my
cool
error

$ echo "${ERR_MSG//[$'\t\r\n']: 0: 3000}"
my
cool
error

$

Reply:
Looks like I over-optimized, it doesn't seem to work when combined with substring :D

Reply:
Reverted to the original solution, thanks for double checking @Michkov